### PR TITLE
chore(ciscospark): Release a new version with deprecation warnings

### PR DESCRIPTION
**Pull request by jeffhoang**

### CODE REVIEW

File: packages/node_modules/@ciscospark/eslint-config-base/README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This package provides Cisco Spark's base JS .eslintrc (without React plugins) as an extensible shared config.
 
+## Status of Module
+This module is deprecated. Please use [@webex/eslint-config-base](https://www.npmjs.com/package/@webex/eslint-config-base) instead.
+
 ## Usage
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/eslint-config-base/README.md]: Let's do what @mattnorris suggested and remove the usage instructions

File: packages/node_modules/@ciscospark/eslint-config-react/README.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 This package provides Cisco Spark's .eslintrc as an extensible shared config.
 
+## Status of Module
+This module is deprecated. Please use [@webex/eslint-config-react](https://www.npmjs.com/package/@webex/eslint-config-react) instead.
+
 ## Usage
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/eslint-config-react/README.md]: Let's do what @mattnorris suggested and remove the usage instructions

File: packages/node_modules/@ciscospark/eslint-config-react/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ciscospark/eslint-config-react",
   "version": "0.6.1",
-  "description": "Cisco Spark's React ESLint config, following our styleguide",
+  "description": "Cisco Spark's React ESLint config, following our styleguide. Note: this module is deprecated.",
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/eslint-config-react/package.json]: Want to add the deprecated keyword on here as well?

**Comment by jeffhoang** [comment]
Updated and squashed commits.

**Pull request merged**